<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Harmony Designer Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/DesignerTool.qrc">
    <normaloff>:/icons16x16/Harmony.png</normaloff>:/icons16x16/Harmony.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="centralVerticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedTabWidgets">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_New_Item">
     <property name="title">
      <string>&amp;New Item</string>
     </property>
     <addaction name="actionNewAudio"/>
     <addaction name="actionNewParticle"/>
     <addaction name="actionNewSprite"/>
     <addaction name="actionNewFont"/>
     <addaction name="separator"/>
     <addaction name="actionNewEntity"/>
    </widget>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="menu_New_Item"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionProjectSettings"/>
    <addaction name="actionLaunchIDE"/>
    <addaction name="actionConnect"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionOpenProject"/>
   <addaction name="actionCloseProject"/>
   <addaction name="separator"/>
   <addaction name="actionProjectSettings"/>
   <addaction name="actionLaunchIDE"/>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetExplorer">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Data Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="explorerContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ExplorerWidget" name="explorer" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetProperties">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Item Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="propertiesContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetOutputLog">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Output Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="outputLogContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="WidgetOutputLog" name="outputLog" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetAtlas">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Atlas Manager</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="atlasContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetAudio">
   <property name="windowTitle">
    <string>Audio Manager</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="audioContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Project-New.png</normaloff>:/icons16x16/items/Project-New.png</iconset>
   </property>
   <property name="text">
    <string>New &amp;Project</string>
   </property>
   <property name="toolTip">
    <string>Create a directory workspace for new project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Project-Open.png</normaloff>:/icons16x16/items/Project-Open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open a directory workspace for an existing project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/edit-copy.png</normaloff>:/icons16x16/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/edit-paste.png</normaloff>:/icons16x16/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remo&amp;ve</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Project-Close.png</normaloff>:/icons16x16/items/Project-Close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close Project</string>
   </property>
   <property name="toolTip">
    <string>Closes the currently active project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/file-exit.png</normaloff>:/icons16x16/file-exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Closes the application</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/file-save.png</normaloff>:/icons16x16/file-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save currently selected item</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/file-saveAll.png</normaloff>:/icons16x16/file-saveAll.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;All</string>
   </property>
   <property name="toolTip">
    <string>Save all open items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNewSprite">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Sprite-New.png</normaloff>:/icons16x16/items/Sprite-New.png</iconset>
   </property>
   <property name="text">
    <string>New Sprite</string>
   </property>
   <property name="toolTip">
    <string>Add a new sprite item in the currently selected project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+S</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-rename.png</normaloff>:/icons16x16/generic-rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionNewFont">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Font-New.png</normaloff>:/icons16x16/items/Font-New.png</iconset>
   </property>
   <property name="text">
    <string>New Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+F</string>
   </property>
  </action>
  <action name="actionNewParticle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Particles-New.png</normaloff>:/icons16x16/items/Particles-New.png</iconset>
   </property>
   <property name="text">
    <string>New Particle Effect</string>
   </property>
   <property name="toolTip">
    <string>Add a new particle effect in the currently selected project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+P</string>
   </property>
  </action>
  <action name="actionNewAudio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Audio-New.png</normaloff>:/icons16x16/items/Audio-New.png</iconset>
   </property>
   <property name="text">
    <string>New Audio</string>
   </property>
   <property name="toolTip">
    <string>Add a new audio bank in the currently selected project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+A</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/debug-notConnected.png</normaloff>
     <normalon>:/icons16x16/debug-connected.png</normalon>:/icons16x16/debug-notConnected.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect and debug a Harmony built game client</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionLaunchIDE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/code.png</normaloff>:/icons16x16/code.png</iconset>
   </property>
   <property name="text">
    <string>Launch IDE</string>
   </property>
   <property name="toolTip">
    <string>Launch the code project in your IDE</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionProjectSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Project-Settings.png</normaloff>:/icons16x16/items/Project-Settings.png</iconset>
   </property>
   <property name="text">
    <string>Project Settings</string>
   </property>
   <property name="toolTip">
    <string>Changes project settings such as the name, initial resolution, assets location, etc.</string>
   </property>
  </action>
  <action name="actionNewEntity">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/items/Entity-New.png</normaloff>:/icons16x16/items/Entity-New.png</iconset>
   </property>
   <property name="text">
    <string>New Entity</string>
   </property>
   <property name="toolTip">
    <string>Add a new entity in the currently selected project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+E</string>
   </property>
  </action>
  <action name="actionTheme_Lappy486">
   <property name="text">
    <string>Lappy 486</string>
   </property>
   <property name="toolTip">
    <string>Now that is a classy computing beige theme</string>
   </property>
  </action>
  <action name="actionTheme_Compe">
   <property name="text">
    <string>Compé</string>
   </property>
   <property name="toolTip">
    <string>So sleek! So-phisticated! So... Compé!</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetOutputLog</class>
   <extends>QWidget</extends>
   <header>WidgetOutputLog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExplorerWidget</class>
   <extends>QWidget</extends>
   <header>ExplorerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/DesignerTool.qrc"/>
 </resources>
 <connections/>
</ui>
